examples/blog-starter-typescript/_posts/dynamic-routing.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: 'Dynamic Routing and Static Generation'
-excerpt: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Praesent elementum facilisis leo vel fringilla est ullamcorper eget. At imperdiet dui accumsan sit amet nulla facilisi morbi tempus.'
```

Comment:
I think all the `facilisi` changes have to be reverted because this is lorem ipsum, not English.